memory/combat_manager.py

```diff
@@ -119,58 +128,76 @@ def _read_battle_commands(self):
 
     # Skill Commands are the menu of the "skills" command (ie Healing Light, Sunball)
     def _read_skill_commands(self):
-        if self.memory.ready_for_updates():
+        if self.memory.ready_for_updates and self.current_encounter_base is not None:
```

Comment:
Getting a bit long, so maybe break the condition out into a temp variable and give it a name.